www/src/templates/template-creator-details.js

```diff
@@ -0,0 +1,324 @@
+                minWidth: `100%`,
+              }}
+              alt={`${creator.name}`}
+              fluid={creator.image.childImageSharp.fluid}
+            />
+          </div>
+          <div
+            css={{
+              display: `flex`,
+              flexDirection: `column`,
+              margin: rhythm(3 / 4),
+              flex: `1`,
+              maxWidth: `720`,
+            }}
+          >
+            {creator.for_hire || creator.hiring ? (
+              <span css={[styles.badge]}>
+                {creator.for_hire ? `Open For Work` : `Hiring`}
+              </span>
+            ) : null}
+            <span
+              css={{
+                display: `flex`,
+                borderBottom: `2px solid black`,
+                alignItems: `center`,
+              }}
+            >
+              <h1
+                css={{
+                  textTransform: `uppercase`,
+                  margin: `0`,
+                }}
+              >
+                {creator.name}
+              </h1>
+              {creator.github && (
+                <GithubIcon
+                  css={{
+                    marginLeft: `auto`,
+                  }}
+                />
+              )}
+            </span>
+            <span
+              css={{
+                borderBottom: `2px solid black`,
+                padding: `${rhythm()} 0`,
+              }}
+            >
+              <p
+                css={{
+                  margin: `0`,
+                }}
+              >
+                {creator.description}
+              </p>
+            </span>
+            <span
+              css={{
+                borderBottom: `2px solid black`,
+                padding: `${rhythm(3 / 4)} 0`,
+                display: `flex`,
+              }}
+            >
+              <p
+                css={{
+                  margin: `0`,
+                  textDecoration: `underline`,
+                  fontWeight: `600`,
+                  width: `150`,
+                }}
+              >
+                Get in touch
+              </p>
+              <a
+                href={creator.website}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {creator.website}
+              </a>
+            </span>
+            <span
+              css={{
+                borderBottom: `2px solid black`,
+                padding: `${rhythm(3 / 4)} 0`,
+                display: `flex`,
+              }}
+            >
+              <p
+                css={{
+                  margin: `0`,
+                  textDecoration: `underline`,
+                  fontWeight: `600`,
+
+                  width: `150`,
+                }}
+              >
+                From
+              </p>
+              <p
+                css={{
+                  margin: `0`,
+                }}
+              >
+                {creator.location}
+              </p>
+            </span>
```

Comment:
For these cases, I'd prefer to see `div`s over `span`s. It's nitpicky, but semantically ideal.